File: Beecrowd/1043.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    float a, b, c;
    cin >> a >> b >> c;

    if(a < (b+c) && b < (a + c) && c < (a + b)){
        cout << fixed << setprecision(1) << "Area = " << ((a+b)*c)/2 << endl;
    } else {
        cout << fixed << setprecision(1) << "Perimetro = " << a+b+c << endl;
    }

    return 0;
}

File: Beecrowd/1043.py
def check(a,b,c):
   return ((a + b) < c) and ((a + c) < b) and ((b + c) < a) 

a,b,c = map(float,input().split())

pode = check(a,b,c)

if pode:
    print(f"Perimetro = {a+b+c:.1f}")
else:
    print(f"Area = {((a+b)*c)/2:.1f}")